doc/howtos/backend/exercise-calendar

```diff
@@ -12,50 +12,49 @@ Index: addons/openacademy/models.py
  from openerp import models, fields, api, exceptions
  
  class Course(models.Model):
-@@ -55,6 +56,8 @@
+@@ -55,6 +56,8 @@ class Session(models.Model):
      attendee_ids = fields.Many2many('res.partner', string="Attendees")
  
      taken_seats = fields.Float(string="Taken seats", compute='_taken_seats')
 +    end_date = fields.Date(string="End Date", store=True,
 +        compute='_get_end_date', inverse='_set_end_date')
  
-     @api.one
      @api.depends('seats', 'attendee_ids')
-@@ -82,6 +85,30 @@
+     def _taken_seats(self):
+@@ -81,6 +84,30 @@ class Session(models.Model):
+                 },
              }
  
-     @api.one
 +    @api.depends('start_date', 'duration')
 +    def _get_end_date(self):
-+        if not (self.start_date and self.duration):
-+            self.end_date = self.start_date
-+            return
++        for r in self:
++            if not (r.start_date and r.duration):
++                r.end_date = r.start_date
++                return
 +
-+        # Add duration to start_date, but: Monday + 5 days = Saturday, so
-+        # subtract one second to get on Friday instead
-+        start = fields.Datetime.from_string(self.start_date)
-+        duration = timedelta(days=self.duration, seconds=-1)
-+        self.end_date = start + duration
++            # Add duration to start_date, but: Monday + 5 days = Saturday, so
++            # subtract one second to get on Friday instead
++            start = fields.Datetime.from_string(r.start_date)
++            duration = timedelta(days=r.duration, seconds=-1)
++            r.end_date = start + duration
 +
-+    @api.one
 +    def _set_end_date(self):
-+        if not (self.start_date and self.end_date):
-+            return
++        for r in self:
++            if not (r.start_date and r.end_date):
++                return
```

Comment:
It should be `continue` instead of `return` (you don't want to exit the function)